src/lib.rs

```diff
@@ -2515,7 +2543,7 @@ impl Connection {
             recv: self.recv_count,
             sent: self.sent_count,
             lost: self.recovery.lost_count,
-            cwnd: self.recovery.cwnd(),
+            cwnd: self.recovery.cc.cwnd(),
```

Comment:
This changes the meaning of the field I think, shouldn't it be be `self.recovery.cwnd_available()`?

Reply:
Assuming stats() is for users to get some stats of the Connection, so providing current cwnd() is better I think (sorry I didn't mention it in the changelog). Is there any reason you want to return cwnd_available()? If someone calls stats() in the middle of connection cwnd() may return a smaller value than real (it returns cwnd-bytes_in_flight, or even usize::MAX).